test/lib/redis-store-spec.js

```diff
@@ -152,6 +168,32 @@ describe('set', function () {
 });
 
 describe('get', function () {
+  it('should return a promise', function (done) {
+    assert.ok(redisCache.get('foo') instanceof Promise);
+    done();
+  });
+
+  it('should resolve promise on success', function (done) {
+    redisCache.set('foo', 'bar')
+      .then(() => redisCache.get('foo'))
+      .then(result => {
+        assert.equal(result, 'bar');
+        done();
+      });
+  });
+
+  it('should reject promise on error', function (done) {
+    var pool = redisCache.store._pool;
+    sinon.stub(pool, 'acquireDb').yieldsAsync('Something unexpected');
+    sinon.stub(pool, 'release');
+    redisCache.get('foo')
+      .catch(() => done())
+      .then(() => {
+        pool.acquireDb.restore();
+        pool.release.restore();
```

Comment:
should test fail if it reaches this part as it is expected to be rejected?

maybe something like this could be clearer:
```js
 var pool = redisCache.store._pool;
 sinon.stub(pool, 'acquireDb').yieldsAsync('Something unexpected');
 sinon.stub(pool, 'release');
 const cleanup = ()=> { 
     pool.acquireDb.restore();
     pool.release.restore()
 }; 
 redisCache.get('foo')
  .then(() => {
    cleanup();
    done(new Error ('Should reject');
   })
  .catch(() =>{
     cleanup();
     done();
   });
```